mpqp/core/instruction/gates/controlled_gate.py

```diff
@@ -39,3 +41,118 @@ def __init__(
         """See parameter description."""
 
         Gate.__init__(self, targets, label)
+
+    def controlled_gate_to_matrix(self) -> Matrix:
```

Comment:
why is this a separate method ? `to_matrix` wouldn't work ?

Reply:
done